install-scripts/install-ubuntu-dependencies.sh

```diff
@@ -18,13 +18,13 @@ sudo apt-get -y update
 sudo apt-get -y install build-essential python-dev swig zlib1g-dev unzip wget
 sudo apt-get -y install wordnet-dev wordnet-sense-index
 sudo apt-get -y install openjdk-9-jdk 
-sudo apt-get -y install ant libcommons-logging-java libslf4j-java liblogback-java libgetopt-java
+sudo apt-get -y install maven libcommons-logging-java libslf4j-java liblogback-java libgetopt-java
```

Comment:
Maven should be enough, all other libraries are installed as maven dependencies
```suggestion
sudo apt-get -y install maven
```

Reply:
Updated.